docs/source/user_resources/additional_information/faq/index.rst

```diff
@@ -0,0 +1,4 @@
+Frequently Asked questions (FAQ)
+================================
```

Comment:
I don't think we need to define `FAQ` as an acronym here since we aren't using it again on this page.
```suggestion
Frequently Asked questions
==========================
```